openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -271,6 +272,25 @@ def load_casting(project_name: str, shot_name: str) -> Set[OpenpypeContainer]:
 
     wait_for_download(project_name, representations)
 
+    return representations
+
+
+def load_casting(project_name: str, shot_name: str) -> Set[OpenpypeContainer]:
+    """Load casting from shot_name using kitsu api.
+
+     Args:
+        project_name (str): Current project name from OpenPype Session.
+        shot_name (str): Current shot name from OpenPype Session.
+
+    Returns:
+        Set[OpenpypeContainer]: Casted assets containers.
+    """
+
+    casting = get_kitsu_casting(project_name, shot_name)
+    assert casting, "Failed to get kitsu casting"
+
+    representations = download_kitsu_casting(project_name, casting)
```

Comment:
I think `download_kitsu_casting` must include `download_kitsu_casting`, I don't see any case where we would need it without getting the casting.

Reply:
Did you mean, it must include `get_kitsu_casting`?

Reply:
I thought I did yes haha